swap/swap_test.go

```diff
@@ -48,6 +49,7 @@ import (
 )
 
 var (
+	loglevel           = flag.Int("logleveld", 2, "verbosity of debug logs")
```

Comment:
I think it's a good idea to keep logLevel like this, but why not just assigning `2` or `3` directly to this variable?

Reply:
I just tried to follow the convention of using flag, as the rest of the test uses this method.
There would be no problem to use directly and int

Reply:
This is correct!